### PR TITLE
How about adding linkedin and Instagram Link as well

### PR DESCRIPTION
**Pull request by Recognizeyourself**
Adding every social media link

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/126"></a>

### CODE REVIEW

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]
Linkedin Spelling is wrong please fix it

**Review by FayasNoushad — APPROVED** [review]

File: src/Components/Links.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ function Links({ links }) {
     youtube: 'red',
     twitter: 'blue',
     github: 'black',
+    linkdin: 'blue'
```
Comment (nhcarrigan) [inline src/Components/Links.js:11]:
```suggestion
    linkedin: 'blue',
```

Fix the lint issue so the build passes, and correct spelling mistake.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
refer to comment 🙂

**Review by BOLT04 — APPROVED** [review]
LGTM 👍

**Comment by nhcarrigan** [comment]
Could you resolve the conflicts please? :)

**Comment by eddiejaoude** [comment]
I resolved the conflict so we can get it merged in 🤓

**Pull request merged**